File: python/prime.py
def isPrime(n):
  for i in range(2, n - 1):
    if n % i == 0:
      return False
  return n > 1

i = 1
found = 0

while found < 3333:
  if isPrime(i):
    found += 1
  i += 1

File: dart/prime.dart
bool isPrime(int n) {
  for (var i = 2; i < n; i++) {
    if (n % i == 0) {
      return false;
    }
  }
  return n > 1;
}

void main () {
  for (var i = 1, found = 0; found < 3333; i++) {
    if (isPrime(i)) {
      found++;
    }
  }
}
